rest-api/edit-conversation-image-title/styles.css

```diff
@@ -0,0 +1,60 @@
+.modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
```

Comment:
It's probably more consistent and easier to understand if you do `right: 0; bottom: 0` instead of setting width and height to 100%